### PR TITLE
Update internal traffic shaping docs

### PR DESCRIPTION
**Pull request by johscheuer**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
This PR corrects/updates the internal documentation for traffic shaping.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/kubelet/dockershim/network/cni/cni.go [hunk 1]
```diff
@@ -87,12 +87,12 @@ type cniBandwidthEntry struct {
 	// IngressRate is the bandwidth rate in bits per second for traffic through container. 0 for no limit. If ingressRate is set, ingressBurst must also be set
 	IngressRate int `json:"ingressRate,omitempty"`
 	// IngressBurst is the bandwidth burst in bits for traffic through container. 0 for no limit. If ingressBurst is set, ingressRate must also be set
```
Comment (mattjmcnaughton) [inline pkg/kubelet/dockershim/network/cni/cni.go]: Nit: Can we capitalize `ingressBurst` and `ingressRate` here?

Same nit for `egressRate` and `egressBurst`.

File: pkg/kubelet/dockershim/network/cni/cni.go [hunk 2]
```diff
@@ -409,11 +409,11 @@ func (plugin *cniNetworkPlugin) buildCNIRuntimeConf(podName string, podNs string
 			// https://github.com/containernetworking/plugins/blob/master/plugins/meta/bandwidth/README.md
 			// Rates are in bits per second, burst values are in bits.
 			bandwidthParam.IngressRate = int(ingress.Value())
-			bandwidthParam.IngressBurst = math.MaxInt32 // no limit
```
Comment (mattjmcnaughton) [inline pkg/kubelet/dockershim/network/cni/cni.go]: Should we add something like... "in practice, limiting to 2Gbit is the equivalent of setting no limit`?

**Review by mattjmcnaughton — APPROVED** [review]
/ok-to-test
/lgtm

Thanks for this pr :)

File: pkg/kubelet/dockershim/network/cni/cni.go (hunk 1 above)
Comment (johscheuer) [reply pkg/kubelet/dockershim/network/cni/cni.go]: Update, thanks for the hint!

File: pkg/kubelet/dockershim/network/cni/cni.go (hunk 2 above)
Comment (johscheuer) [reply pkg/kubelet/dockershim/network/cni/cni.go]: Yeah, sounds reasonable.

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm

Still good in my book - thanks for this documentation improvement.

**Comment by johscheuer** [comment]
Friendly ping @dcbw and @nicksardo  :)

**Comment by johscheuer** [comment]
Ping @dcbw and @nicksardo :)

**Comment by johscheuer** [comment]
Friendly ping again @dcbw and @nicksardo :)

**Comment by danwinship** [comment]
we should at least update the docs to explain the alpha annotations but it would be good to push them forward to GA too...

**Comment by johscheuer** [comment]
@danwinship I can update the docs.

I just wondered if the annotations shouldn't actually be something like this `networking.alpha.kubernetes.io/ingress-bandwidth: 1M` (oriented on other annotations of Kubernetes) rather than the current annotation `kubernetes.io/ingress-bandwidth` ?

**Comment by johscheuer** [comment]
It seems like the approve command didn't work, right ?

**Comment by johscheuer** [comment]
ping @dcbw is there something still missing?

**Comment by derekwaynecarr** [comment]
/assign
/approve

note: bandwidth shaping is not officially a supported feature of kubernetes.

**Pull request merged**